clicommand/pipeline_upload.go

```diff
@@ -142,7 +143,11 @@ var PipelineUploadCommand = cli.Command{
 			Value:  "origin/main",
 			EnvVar: "BUILDKITE_GIT_DIFF_BASE,BUILDKITE_PULL_REQUEST_BASE_BRANCH",
 		},
-
+		cli.BoolFlag{
+			Name:   "validate-dependencies",
+			Usage:  "When true, validates the dependencies of each step when uploading.",
+			EnvVar: "BUILDKITE_PIPELINE_VALIDATE_DEPENDENCIES",
```

Comment:
The env vars in this command don't seem to follow the same pattern, so I just chose one. Alternatively: BUILDKITE_AGENT_PIPELINE_UPLOAD_VALIDATE_DEPENDENCIES ?

Reply:
`BUILDKITE_PIPELINE_VALIDATE_DEPENDENCIES` sounds OK to me.